backend/templates/model-na-31-2.typ

```diff
@@ -0,0 +1,351 @@
+
+#table(
+  columns: (auto, 1fr, auto),
+  rows: 13,
+  gutter: 10pt,
+  stroke: 0.5pt + black,
+  table.header(
+    table.cell(stroke: none, text(size: 8pt, style: "italic", [Voorletters])),
+    table.cell(stroke: none, text(size: 8pt, style: "italic", [Achternaam leden #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam]])),
+    table.cell(stroke: none, text(size: 8pt, style: "italic", [Aanwezig (van - tot)])),
+  ),
+  ..range(0, 12).map(_ => {(
+    table.cell()[],
+    table.cell(inset: 0pt, grid(rows: 2, columns: 2, inset: 6pt, grid.vline(x: 1, stroke: 0.5pt + black),
+      grid.cell(text(size: 8pt, [Achternaam])),
+      grid.cell()[],
+      grid.cell(text(size: 8pt, [Locatie])),
+      grid.cell()[]
+    )),
+    table.cell(align: horizon, stack(dir: ltr, spacing: 3pt, time_input(time: ""), align(top, [-]), time_input(time: ""))),
+  )}).flatten(),
+)
+
+#pagebreak(weak: true)
+
+= Stembureaus binnen #is_municipality[de gemeente][het openbaar lichaam]
+
+#emph[Overzicht van de nummers en locaties van alle stembureaus binnen #is_municipality[de gemeente][het openbaar lichaam]:]
+#columns(2, gutter: 10pt)[
+  #table(
+    columns: (auto, 1fr),
+    inset: 8pt,
+    stroke: 0.5pt + black,
+    table.header(
+      [_Nummer_], [_Locatie stembureau_]
+    ),
+    ..input.polling_stations.map(polling_station => {(
+      [#polling_station.number],
+      [
+        #if polling_station.polling_station_type == "Mobiel" [
+          _(Mobiel stembureau)_
+        ] else [
+          #polling_station.street #polling_station.house_number #if polling_station.house_number_addition != none [
+            #polling_station.house_number_addition
+          ] \
+          #polling_station.postal_code #polling_station.locality
+        ]
+      ]
+    )}).flatten(),
+  )
+]
+
+#pagebreak(weak: true)
+
+= Aantal toegelaten kiezers zoals vastgesteld door het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam]
+
+#emph[Som van de aantallen toegelaten kiezers van alle stembureaus in #is_municipality[de gemeente][het openbaar lichaam]. Dit is de som van de
+aantallen opgenomen in bijlage 2, rubriek 1 of, indien van toepassing, rubriek 3.]
+
+#summary_table(
+  (
+    ([Aantal geldige stempassen], [A], mono[#input.summary.voters_counts.poll_card_count]),
+    ([Aantal geldige volmachtbewijzen (schriftelijk of via ingevulde stem- of kiezerspas)], [B], mono[#input.summary.voters_counts.proxy_certificate_count]),
+    ([Aantal geldige kiezerspassen], [C], mono[#input.summary.voters_counts.voter_card_count]),
+  ),
+  added_together: [Tel op +],
+  total: ([Het aantal tot de stemming toegelaten kiezers ( A + B + C = D )], [D], mono[#input.summary.voters_counts.total_admitted_voters_count])
+)
+
+= Aantal getelde stembillettenuitgebrachte stemmen zoals vastgesteld door het #is_municipality[gemeentelijk stembureau][stembureau voor het openbaar lichaam]
```

Comment:
I was wondering how much freedom we have in changing the text from these models, I would very much prefer to use the suggestions though if we have that freedom.

Reply:
I got the suggestions from the current model, see https://wetten.overheid.nl/afbeelding?toestandid=BWBR0034180/2023-11-01_0&naam=271034.png&schalen=nee

Reply:
Interesting, I used the file from https://www.rijksoverheid.nl/documenten/publicaties/2022/11/18/model-na-31-2 as my starting point

Reply:
What the heck, that's weird. Also this `stembillettenuitgebrachte stemmen` just makes no sense in that text. There should at least be a `/` in between :confused: @jorisleker @praseodym @jschuurk-kr Any idea what file is correct?

Reply:
@arjen-ba What's the best reference/source for the latest version of the "modellen"?

Reply:
> Interesting, I used the file from https://www.rijksoverheid.nl/documenten/publicaties/2022/11/18/model-na-31-2 as my starting point

Looks like they just made an error in the PDF, because the ODT is correct compared to the wetten.overheid.nl file and the suggestion I made.

Reply:
> @arjen-ba What's the best reference/source for the latest version of the "modellen"?

https://www.rijksoverheid.nl/onderwerpen/verkiezingen/documenten/publicaties/2022/11/18/model-na-31-2

(the one dated 29-04-2024 is the correct one, so it's the same one as in this (different) link)

Reply:
> I was wondering how much freedom we have in changing the text from these models, I would very much prefer to use the suggestions though if we have that freedom.

There is an ongoing project to revise these documents, so wouldn't put too much effort into changing things now. There are much bigger changes coming, so as @praseodym also mentioned: no need to be 100% accurate right now. And textual changes/improvements you make will probably become obsolete.

Reply:
I've updated the title here to conform to the odt